qcodes_contrib_drivers/drivers/ERAInstruments/erasynth.py

```diff
@@ -0,0 +1,1014 @@
+"""Driver for the ERASynth/ERASynth+/ERASynth++ signal generators.
+
+Author: Victor Negîrneac, vnegirneac@qblox.com
+
+For official instrument support visit:
+
+- https://erainstruments.com
+- https://github.com/erainstruments/
+
+NB this driver adds parameters using the new `@add_paramter` decorator-style.
+See QCoDeS docs on writing drivers for more details.
+"""
+from __future__ import annotations
+
+from typing import Union
+import time
+import json
+from qcodes import Instrument
+from qcodes import validators
+from qcodes.instrument.base import InstanceAttr, add_parameter
+import serial  # pip install pyserial
+from serial.tools.list_ports import comports  # pip install pyserial
```

Comment:
something like this:
```suggestion
try:
    import serial  # pip install pyserial
    from serial.tools.list_ports import comports  # pip install pyserial
except ImportError as e:
    raise ImportError("Failed to import ``pyserial`` package that is required for this driver, install it with ``pip install pyserial`` and try again") from e
```